test/unit/toggle/toggle-component.js

```diff
@@ -0,0 +1,151 @@
+import Toggle from '../../../src/components/toggle';
+import ToggleView from '../../../src/views/toggle';
+
+const fakeClient = {
```

Comment:
Do we need the fakeClient? The toggle component doesn't seem to be accessing the client from the props.